Resolve Typo

prashant-645 opened the pull request:
### **Description:**
Resolved Typo issue

### 
**Fixed Issue: (if Relevant )**
**expected result :** 
It can be used to set width for the whole page wrapper or for the page elements _individually_ like header, footer, and so on. on line 364;

**Actually Result:** 
It can be used to set width for the whole page wrapper or for the page elements _individualy_ like header, footer, and so on. on line 364;

### Contribution checklist (*)
 - [X] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

prashant-645 committed:
Resolve Typo

Corrected the spelling of individually from individualy on line no. 364.

magento-cicd2 commented:
[![CLA assistant check](https://cla.dev.magento.com/pull/badge/not_signed)](https://cla.dev.magento.com/magento/magento2?pullRequest=22624) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2?pullRequest=22624) before we can accept your contribution.<br/>

prashant-645 committed:
Resolved Typo error

At both places the spelling of padding was paddings
updated line no. 301 and 82

magento-engcom-team commented:
@prashantsharmacedcoss thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.
